sql/atxs/atxs.go

```diff
@@ -844,3 +876,40 @@ func PrevATXCollisions(db sql.Executor) ([]PrevATXCollision, error) {
 
 	return result, nil
 }
+
+func Units(db sql.Executor, atxID types.ATXID, nodeID types.NodeID) (uint32, error) {
+	var units uint32
+	rows, err := db.Exec(`
+		SELECT units FROM posts WHERE atxid = ?1 AND pubkey = ?2;`,
+		func(stmt *sql.Statement) {
+			stmt.BindBytes(1, atxID.Bytes())
+			stmt.BindBytes(2, nodeID.Bytes())
+		},
+		func(stmt *sql.Statement) bool {
+			units = uint32(stmt.ColumnInt64(0))
+			return false
+		},
+	)
+	if rows == 0 {
+		return 0, sql.ErrNotFound
+	}
+	return units, err
+}
+
+func SetUnits(db sql.Executor, atxID types.ATXID, units map[types.NodeID]uint32) error {
```

Comment:
This is inconsistent with the function above. Either we also set each `numUnit` in isolation:

```suggestion
func SetUnit(db sql.Executor, atxID types.ATXID, nodeID types.NodeID, numUnits uint32) error {
```

or we return `Units` as `map[types.NodeID]uint32`. Wdyt?

Reply:
I will change `SetUnits` to set units for 1 ID